app/models/connector/api_1802/request/CreateAmendSEAnnualSubmissionRequestData.scala

```diff
@@ -16,9 +16,42 @@
 
 package models.connector.api_1802.request
 
+import cats.implicits.catsSyntaxOptionId
 import models.common.{BusinessId, Nino, TaxYear}
+import models.connector.api_1803
+import models.connector.api_1803.AnnualNonFinancialsType
+import models.frontend.nics.NICsClass4Answers
 
 case class CreateAmendSEAnnualSubmissionRequestData(taxYear: TaxYear,
                                                     nino: Nino,
                                                     businessId: BusinessId,
                                                     body: CreateAmendSEAnnualSubmissionRequestBody)
+
+object CreateAmendSEAnnualSubmissionRequestData {
+  def mkNicsClassFourSingleBusinessRequestBody(answers: NICsClass4Answers,
+                                               existingAnswers: api_1803.SuccessResponseSchema): CreateAmendSEAnnualSubmissionRequestBody = {
+//    val maybeBusinessDetailsChangedRecently = existingAnswers.annualNonFinancials.flatMap(_.businessDetailsChangedRecently)
+
+//    val annualNonFinancialsAnswers = AnnualNonFinancials(maybeBusinessDetailsChangedRecently, answers.class4NICs.some, answers.class4ExemptionReason.map(_.exemptionCode))
+    val annualNonFinancialsAnswers = AnnualNonFinancials(Some(true), answers.class4NICs.some, answers.class4ExemptionReason.map(_.exemptionCode))
+    // TODO SASS-8728 businessDetailsChangedRecently is a compulsory value, where does it come from?
```

Comment:
businessDetailsChangedRecently in not compulsory in the yaml but API fails if sent None. Which is correct? Where can we get the answer from? TBC